heron/scheduler-core/src/java/com/twitter/heron/scheduler/RuntimeManagerRunner.java

```diff
@@ -157,8 +165,12 @@ void killTopologyHandler(String topologyName) throws TopologyRuntimeManagementEx
     // clean up the state of the topology in state manager
     cleanState(topologyName, Runtime.schedulerStateManagerAdaptor(runtime));
 
-    // Clean the connection when we are done.
-    LOG.fine("Scheduler killed topology successfully.");
+    if (foundData) {
+      LOG.fine(String.format("Scheduler killed topology %s successfully.", topologyName));
+    } else {
+      LOG.warning(String.format("Topology %s data is not found, but scheduler has tried to"
```

Comment:
What is this message for?

Reply:
I think if topology data is not available, it is possible that the topology was not running.  The message "Scheduler killed topology %s successfully."  can be misleading. So I think in this case, "scheduler has tried to clean up" is more accurate.